src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -1296,7 +1359,27 @@ public Result startStreaming(Broadcast broadcast)
 				broadcast.getType().equals(AntMediaApplicationAdapter.STREAM_SOURCE) ||
 				broadcast.getType().equals(AntMediaApplicationAdapter.VOD)
 				)  {
```

Comment:
Please all check that if there is enough resource(`statsCollector.enoughResource()`) available in the instance because it may overload the systems unexpectedly in cluster mode or standalone mode

Reply:
done